Ticket-booking-spring-app/src/main/java/com/example/demo/rest/SeatAvailabilityRestController.java

```diff
@@ -0,0 +1,26 @@
+package com.example.demo.rest;
+
+import com.example.demo.entity.SeatAvailability;
+import com.example.demo.service.seat_availability.SeatAvailabilityService;
+import com.example.demo.util.SeatAvailabilityResponse;
+import org.springframework.web.bind.annotation.GetMapping;
+import org.springframework.web.bind.annotation.PathVariable;
+import org.springframework.web.bind.annotation.RequestMapping;
+import org.springframework.web.bind.annotation.RestController;
+
+import java.util.List;
+
+@RestController
+@RequestMapping("/seat_availability_api")
+public class SeatAvailabilityRestController {
+    private SeatAvailabilityService seatAvailabilityService;
+
+    public SeatAvailabilityRestController(SeatAvailabilityService theSeatAvailabilityService) {
```

Comment:
Nietypowe nazewnictwo zmiennych. Raczej robimy 1-1 a w konstruktorze dajemy `this`